Exiled.Events/Handlers/Internal/Round.cs

```diff
@@ -25,7 +25,10 @@ public static void OnWaitingForPlayers()
             MultiAdminFeatures.CallEvent(MultiAdminFeatures.EventType.WAITING_FOR_PLAYERS);
 
             if (Events.Instance.Config.ShouldReloadConfigsAtRoundRestart)
```

Comment:
Shouldn't we create a new config for reloading translations when the round restarts?
Like `ShouldReloadTranslationsAtRoundRestart`.